### PR TITLE
PLANET-7138: Remove text shadows on links

### PR DESCRIPTION
**Pull request by GP-Dan-Tovbein**
Ref: https://jira.greenpeace.org/browse/PLANET-7138

## Description
The idea comes from [here](https://github.com/greenpeace/planet4-master-theme/pull/1996#discussion_r1174932194)

Demo page: https://www-dev.greenpeace.org/test-proteus/test-text-links-without-shadow/

### CODE REVIEW

**Comment by planet-4** [comment]
### Test instance is ready :rocket:

:new_moon: [proteus](https://www-dev.greenpeace.org/test-proteus) | [admin](https://www-dev.greenpeace.org/test-proteus/wp-admin/) | [blocks report](https://www-dev.greenpeace.org/test-proteus/wp-admin/?page=plugin_blocks_report) | [CircleCI](https://app.circleci.com/pipelines/github/greenpeace/planet4-test-proteus) | [composer-local.json](https://github.com/greenpeace/planet4-test-proteus/blob/main/composer-local.json)

:watch: 2023.05.02 15:25:38

File: assets/src/styles/blocks/Covers/styles/CampaignCovers.scss [hunk 1]
```diff
@@ -82,7 +82,7 @@
 
         &:hover {
           &:before {
-            opacity: 0.1;
+            opacity: 0.3;
```
Comment (GP-Dan-Tovbein) [inline assets/src/styles/blocks/Covers/styles/CampaignCovers.scss:85]: Same as [here](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/1044/files#r1182905575)

**Review by mleray — COMMENTED** [review]
I see on the demo page that when the new identity styles are enabled, then 2 different yellow colors are used, isn't that weird? Don't we want to use the same yellow in both blocks? 🤔

File: assets/src/styles/blocks/Covers/styles/CampaignCovers.scss [hunk 2]
```diff
@@ -97,15 +97,14 @@
         text-align: center;
         font-size: $font-size-md;
         font-family: var(--headings--font-family);
-        color: $yellow;
+        color: var(--p4-action-yellow-500, $yellow);
```
[the remark this answers is not in the document]
Comment (mleray) [reply assets/src/styles/blocks/Covers/styles/CampaignCovers.scss:100]: This color `--p4-action-yellow-500` should only be available with the new identity, if I'm not mistaken? Maybe we should instead create a CSS variable like `--yellow-tag--color` or something like that with `$yellow` as default value and then set it to `--p4-action-yellow-500` only with the new identity?

**Comment by GP-Dan-Tovbein** [comment]
> I see on the demo page that when the new identity styles are enabled, then 2 different yellow colors are used, isn't that weird? Don't we want to use the same yellow in both blocks? 🤔

@mleray FYI, Houssam decided to use the `--p4-action-yellow-500` to all these links. And that's why I thought that it’s good to keep the same yellow color, but adding a new CSS variable to each variant, and finally override that variables within the **new-identity stylesheet**.

**Review by mleray — APPROVED** [review]
Looks good to me 👍

**Pull request merged**